src/transformers/trainer_tf.py

```diff
@@ -91,10 +91,11 @@ def __init__(
         if is_wandb_available():
             self._setup_wandb()
         else:
-            logger.info(
-                "You are instantiating a Trainer but W&B is not installed. To use wandb logging, "
-                "run `pip install wandb; wandb login` see https://docs.wandb.com/huggingface."
-            )
+            if os.environ.get("WANDB_DISABLED") != "true":
```

Comment:
Same here

Reply:
Thank you, @sgugger, for the suggestions - done.